lib/dataspec/src/lib.rs

```diff
@@ -20,15 +17,15 @@ pub struct Connections {
     pub timeout: u64,
 }
 
-#[derive(Copy, Clone, Serialize, Deserialize)]
+#[derive(Default, Deserialize, Serialize)]
 pub struct Requests {
```

Comment:
Why did you get rid of copy + clone on these?

Reply:
Mostly because it didn't seem to be used and I didn't want us to accidentally trigger a copy instead of a move. Seems like all of them should have the same copy vs move semantics.